README.md

```diff
@@ -7,33 +7,52 @@ Command line tool to upload/manage project on the [duke-data-service](https://gi
 
 
 # Requirements
-- [python](https://www.python.org/) - version 2.7+ with a functional ssl module. 
+
+- [python](https://www.python.org/) - version 2.7+ with a functional ssl module.
 - [requests](http://docs.python-requests.org/en/master/) - python module
 - [PyYAML](http://pyyaml.org/wiki/PyYAML) - python module
 
 The preferred python versions are 2.7.9+ or 3.4.1+ as they have functional ssl modules by default.
 Older python 2.7 may work by following this guide: [Older-python-2.7-setup](https://github.com/Duke-GCB/DukeDSClient/wiki/Older-python-2.7-setup)
 
-# Install or Upgrade:
+# Installation:
+
+DukeDSClient is written in Python and packaged for easy installation from [PyPI](https://pypi.org/project/DukeDSClient/), the Python Package Index. We recommend installing into a Python [virtual environment](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments).
+Please see [the tutorial on installing packages](https://packaging.python.org/tutorials/installing-packages/) for full details, but the below commands will create a virtual environment named **ddsclient-env** and install **DukeDSClient**:
```

Comment:
Add a note here that if one does not have superuser or admin privileges on the machine, one will have to use this mechanism, or [supply `--user` to `pip`](https://docs.python.org/3/install/index.html#alternate-installation-the-user-scheme)? (Or use a different link out if there's better documentation on this elsewhere.)